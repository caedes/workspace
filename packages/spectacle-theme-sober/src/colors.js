export default {
  primary: 'white',
  secondary: '#1F2022',
  tertiary: '#03A9FC',
  quarternary: '#CECECE',
};
