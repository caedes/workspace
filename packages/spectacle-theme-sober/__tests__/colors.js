import colors from '../src/colors';

describe('colors', () => {
  it('returns the correct snapshot', () => {
    expect(colors).toMatchSnapshot();
  });
});
