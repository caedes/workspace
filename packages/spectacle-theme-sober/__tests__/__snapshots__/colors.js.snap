// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors returns the correct snapshot 1`] = `
Object {
  "primary": "white",
  "quarternary": "#CECECE",
  "secondary": "#1F2022",
  "tertiary": "#03A9FC",
}
`;
